Modules/Bridge/NumPy/include/itkPyBuffer.hxx

```diff
@@ -94,9 +92,19 @@ PyBuffer<TImage>::_GetImageViewFromArray(PyObject * arr, PyObject * shape, PyObj
     return nullptr;
   }
 
-  if (isFortranContiguous)
+  if (PyBuffer_IsContiguous(&pyBuffer, 'C') == 0)
   {
-    std::reverse(size.begin(), size.end());
+    if (PyBuffer_IsContiguous(&pyBuffer, 'F') == 1)
+    {
+      // The buffer is Fortran contiguous (and not C-contiguous), so reverse the size elements.
+      std::reverse(size.begin(), size.end());
+    }
+    else
+    {
+      // The buffer is neither Fortran nor C-contiguous. This is unlikely to happen, because PyBUF_ANY_CONTIGUOUS was
+      // specified as argument, when retrieving the buffer. But if it _does_ happen, it's obviously an error.
+      PyErr_SetString(PyExc_RuntimeError, "The buffer should be contiguous, but it is not!");
```

Comment:
Here `return nullptr;` should be added to "throw" the CPython error.

Reply:
Good catch, thanks @thewtex ! Should be adressed by the [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/bd31ed47cc00047e3f2d28c587ae2acc5dd1116b..b769b81f648ecabf80a481f31ee44813ea4ea4dd) that I just did, please check!